src/handwritten/__init__.py

```diff
@@ -0,0 +1,13 @@
+from nidaqmx.errors import DaqError, DaqReadError, DaqWriteError, DaqWarning, DaqResourceWarning
```

Comment:
should the files in the other folder "nidaqmx" be removed? So it shows as moved, but not additions?

Reply:
I think the removal of the original nidaqmx code from the source folder can be done once we have generated all nidaqmx files and the existing tests in the repo is passed.

Reply:
Yep, we're gonna wait to do that until all the codegen is done